### PR TITLE
SD-2463 As a user I want to see the item history - backend operations

### PR DESCRIPTION
**Pull request by mugurrus**
Issues:
- ITEM_RESTORE - the client uses PATCH instead of REPLACE
- did not implement ITEM_LOCK, ITEM_UNLOCK and ITEM_ADD_TO_PACKAGE because these operations do not create a new document version

### CODE REVIEW

**Comment by ioanpocol** [comment]
:+1:

**Comment by sivakuna-aap** [comment]
Needs rebasing...

File: server/apps/prepopulate/app_prepopulate_data.json [hunk 1]
```diff
@@ -201,6 +201,82 @@
...
+		                "X", 
+		                "T"
+		            ]
+		        },
+		        "pubstatus" : {
+		            "type": "string",
+		            "required" : true,
+		            "allowed": ["usable"]
+		        },
+		        "destination_groups" : {
```
Comment (amagdas) [inline server/apps/prepopulate/app_prepopulate_data.json]: Destination groups and output channels were removed.

File: client/app/scripts/superdesk-authoring/authoring.js [hunk 2]
```diff
@@ -900,11 +900,12 @@
...
-                    .then(function() {
-                        var p = lock.unlock($scope.origItem);
-                        return p;
-                    });
+                    return lock.unlock($scope.origItem);
+                    // return $scope.save()
+                    // .then(function() {
+                    //     var p = lock.unlock($scope.origItem);
+                    //     return p;
+                    // });
```
Comment (plamut) [inline client/app/scripts/superdesk-authoring/authoring.js]: Probably best to remove this if not needed anymore?

**Comment by ioanpocol** [comment]
Created PR #794

**Pull request closed without merging**